ls1intum/Artemis
# Fix the participation header overshooting

### Checklist

### Motivation and Context
Before
![grafik](https://user-images.githubusercontent.com/72132281/125068573-eb850d80-e0b5-11eb-8825-c01dc0db0c55.png)

After
![grafik](https://user-images.githubusercontent.com/72132281/125068522-dc05c480-e0b5-11eb-963f-c3ea683bfc05.png)

### Description

The default padding changed to 1rem, so we adjust this in the SCSS.

### Steps for Testing

1. Log in to Artemis
2. Open a modeling exercise for example
3. Check that the header aligns with the card body (see screenshots)
